cmd/up/syncthing.go

```diff
@@ -177,13 +176,14 @@ func (up *upContext) synchronizeFiles(ctx context.Context) error {
 	}()
 
 	if err := up.Sy.WaitForCompletion(ctx, reporter); err != nil {
-		analytics.TrackSyncError()
+		up.analyticsMeta.AddErrSync()
 		switch err {
 		case oktetoErrors.ErrLostSyncthing:
 			return err
 		case oktetoErrors.ErrInsufficientSpace:
 			return up.getInsufficientSpaceError(err)
 		case oktetoErrors.ErrNeedsResetSyncError:
+			up.analyticsMeta.AddErrResetDatabase()
```

Comment:
i had the same feeling, this is something we already where tracking with the ResetDatabase event, which just sent boolean when happened, in order to track it into the event i add the prop as i understood it caused an error :/ 

in terms of error in the cmd, we are just tracking if is successful or not, with just some kind of errors. I think it would be interesting to track why if failed in every scenario but we need to refactor the code more, wdyt?

Reply:
I'd add it. This way we'll have all the sync errors that we know

Reply:
i added them to the follow up PR with new props as this was refactor of existent ones

https://github.com/okteto/okteto/pull/3876/commits/9489d017bfa4627f4205a2acf95a6ac7a0a7db93